Makefile

```diff
@@ -38,6 +38,23 @@ load-test:
 	kubectl apply -f ./test/load/assets/templates/provider.yaml
 	kube-burner init -c ./test/load/cfg.yml --metrics-profile ./test/load/metrics.yml
 
+#Run tests for subfiles
```

Comment:
@prakrit55 this is definitely the way to go 🚀 however I think it would make sense to also have a single command such us unit-test and lint that calls all of the commands together

Reply:
also please test the commands, before each $(MAKE) there must be a tab not spaces otherwise the make file will fail other pipelines 😄 https://github.com/keptn/lifecycle-toolkit/actions/runs/6185297782/job/16798393518?pr=2101#step:4:29

Reply:
hey @RealAnna, thanks for the clarification